lib/mapObjects/CGHeroInstance.h

```diff
@@ -102,20 +102,6 @@ class DLL_LINKAGE CGHeroInstance : public CArmedInstance, public IBoatGenerator,
 		}
 	} patrol;
 
-	// deprecated - used only for loading of old saves
```

Comment:
Just letting you know that I also removed this struct in H3M branch, may lead to conflict unless git manages to resolve it

Reply:
You did it as separate commit? If so, I would take this commit as part of rebase, then git will  ignore it after a merge for sure.

Reply:
No, it is not inside separate commit. @IvanSavenko, can you take this commit inside your H3M branch?